spec/ws/style_spec.rb

```diff
@@ -75,8 +75,9 @@ def not_ci_file?(diff)
   end
 
   it 'has changelog entry for current version' do
-    parser = Keepachangelog::MarkdownParser.load('CHANGELOG.md')
-
-    expect(parser.parsed_content['versions'].keys).to include(start_with(head_version))
+    parser = ParseAChangelog.parse('CHANGELOG.md')
+    versions = parser.elements[2].elements.map { |element| element.elements[1].text_value }
+    first_version = versions[0]
+    expect(first_version).to include(head_version)
```

Comment:
Now that I've thought about it, you could probably just do `expect(versions.first).to include(head_version)` here. It's a little more concise and provides the same semantics as creating the variable.